manager-bundle/src/ContaoManager/Plugin.php

```diff
@@ -343,6 +343,28 @@ private function checkMailerTransport(array $extensionConfigs, ContainerBuilder
         return $extensionConfigs;
     }
 
+    /**
+     * Dynamically adds a default mailer to the config, if no mailer is defined.
```

Comment:
Why do we need this?

Reply:
You mean as opposed to adding it to the skeleton `config.yml`? Because the config needs to either look like this:

```yml
framework:
  mailer:
    dsn: …
```

or this:

```yml
framework:
  mailer:
    transports:
      foo_transport: …
```

you can't have both, i.e.

```yml
framework:
  mailer:
    dsn: …
    foo_transport: …
```

will result in an error when validating the config. Thus we need to add the default mailer dynamically.

Reply:
If someone wants to add their own transport, they can set `framework.mailer.dsn: ~`, can't they?

Reply:
Yes, but if someone adds

```yml
# config/config.yml
framework:
  mailer:
    foo_transport: …
```

and Contao defines

```yml
framework:
  mailer:
    dsn: …
```

in `Resources/skeleton/config/config.yml` then it leads to an error.

Reply:
I see.

Reply:
Should I add this to the comment as well?

Reply:
Yes please.